testng-core/src/test/java/test/GraphTest.java

```diff
@@ -42,8 +42,6 @@ public void sort() {
     Assert.assertTrue("2".equals(l.get(i)) || "2.1".equals(l.get(i)) || "2.2".equals(l.get(i)));
     i++;
     Assert.assertEquals("3", l.get(i));
-
-    Assert.assertEquals(1, g.getIndependentNodes().size());
```

Comment:
Why did we have to remove this assertion ?

Reply:
because I removed the IndependentNodes, which skipped the second sort for all methods.

Reply:
@martinaldrin - Is that removal required for this change to be implemented? If not, I would prefer that we revert the change.

Reply:
@krmahadevan I think we can remove internal methods when they are no more used.